.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "deps/lexy"]
+	path = deps/lexy
+	url = git@github.com:foonathan/lexy.git
```

Comment:
Will this work for users not using SSH? The main repo uses HTTPS URLs for submodules, so following it's style in this case it would be `https://github.com/foonathan/lexy`.